hera_cal/delay_filter.py

```diff
@@ -82,71 +80,6 @@ def run_filter(self, to_filter=None, weight_dict=None, horizon=1., standoff=0.15
             if write_cache:
                 filter_cache = io.write_filter_cache_scratch(filter_cache, cache_dir, skip_keys=keys_before)
 
-    def get_filled_data(self):
```

Comment:
also, is there a good reason to move this elsewhere? this will break a lot of existing pipeline. if you want to move the core code elsewhere that's fine, but I think we should leave shallow wrappers here with their docstrings for consistency.

Reply:
I decided to move this here because it has functionality that is useful for all kinds of filtering (delay/fringe-rate domain) and since VisClean is the base class for this functionality I think it makes sense for it to be available for all code inheriting VisClean.

Reply:
I'm not sure how this breaks the pipeline since `DelayFilter` inherits `VisClean` so anything that was calling this for a `DelayFilter` object should still be able to do so.

Reply:
ah you are right. yeah I thought some of the stuff in `get_filled_data` was specific to DelayFilter, but it is all consistent with functionality in `visclean` so this is probably fine!

Reply:
Cool :)